python/doc/theory/meta_modeling/cross_validation.rst

```diff
@@ -1,186 +1,654 @@
+
+for :math:`j = 1, ..., n`.
+The Monte-Carlo estimator of the mean squared error is:
+
+.. math::
+    \widehat{\operatorname{MSE}}\left(\metaModel\right)
+    = \frac{1}{n} \sum_{j = 1}^n \left( y^{(j)} - \metaModel \left(\vect{x}^{(j)}\right) \right)^2
+
+The previous equation can be equivalently expressed depending on
+the model since :math:`y^{(j)} = \model\left(\vect{x}^{(j)}\right)`.
+It seems, however, more consistent to use :math:`y^{(j)}` because the
+true model :math:`g` is unknown (otherwise we would not use a
+surrogate).
+
+The sample relative mean squared error is:
+
+  .. math::
 
-**K-fold cross-validation error estimate**
+      \widehat{\varepsilon}_{MSE}\left(\metaModel\right)
+      = \frac{\widehat{\operatorname{MSE}}\left(\metaModel\right)}{\widehat{\sigma}^2(Y)}
 
-The :math:`K`-fold cross-validation technique relies on splitting the
-data set :math:`\cX` into :math:`K` mutually exclusive sub-samples
-:math:`\cX_1, \dots, \cX_K` of approximately equal size. A sub-sample
-:math:`\cX_i`, :math:`i \in \{ 1, \dots, K\}` is set aside, then the
-response surface :math:`\widehat{h}^{(-\cX_i)}` is built on the
-remaining sub-samples :math:`\cX \setminus \cX_i`. The approximation
-error is estimated on the set-aside sample :math:`\cX_i`:
+where :math:`\widehat{\sigma}^2(Y)` is the sample variance of the random output:
 
 .. math::
-   Err^{(i)}  = \dfrac{1}{ |\cX_i|}  \sum\limits_{\underline{x}^{(j)} \in \cX_i} \left[ h(\underline{x}^{(j)}) - \widehat{h}^{(-\cX_i)} {(\underline{x}^{(j)})} \right]^2
 
-in which
-:math:`h(\underline{x}^{(j)}) - \widehat{h}^{(-\cX_i)} {(\underline{x}^{(j)})}`
-is the *predicted residual* defined as the difference between the
-evaluation of :math:`h(\cdot)` and the prediction with
-:math:`\widehat{h}^{(-\cX_i)}(\cdot)` at :math:`\underline{x}^{(j)}`
-in the sub-sample :math:`\cX_i` whose cardinality is :math:`|\cX_i|`.
+      \widehat{\sigma}^2(Y)
+      = \frac{1}{n - 1} \sum_{j = 1}^n \left( y^{(j)} -   \bar{y} \right)^2
+
+where :math:`\bar{y}` is the sample mean of the output:
+
+.. math::
 
-The approximation errors :math:`Err^{(i)}` are estimated with each of
-the sub-samples :math:`\cX_i`, :math:`i \in \{ 1, \dots, K\}` being
-used as the validation set whereas the remaining sub-samples
-:math:`\cX \setminus \cX_i` being used for training. Finally the
-:math:`K`-fold cross-validation error estimate is obtained as the
-average:
+    \bar{y} = \frac{1}{n} \sum_{j = 1}^n y^{(j)}.
+
+If the test set :math:`\set{D}` is not independent from the training set
+(the set used to calibrate the metamodel), then the previous estimator
+may underestimate the true value of the mean squared error.
+In order to create a test set independent from the training set, a
+simple method is to split the data set into two parts.
+The drawback of this method is that this reduces the size of the training
+set, so that the mean squared error evaluated on the test set can be pessimistic.
```

Comment:
The particular sentence that you point means that splitting the data into two non overlapping sub-samples reduces the amount of data available for both tasks:
- The sub-sample size used for training is smaller. This implies that the accuracy of the metamodel is lower than the accuracy of the metamodel trained on the full data, which is what we are interested in. Hence, the coefficients estimated from the sub-sample are less accurately estimated. Hence, the R2 score is most of the times pessimistic.
- The sub-sample size used for testing is smaller. This implies that the estimated R2 score is less accurately estimated. This reduced-quality estimate of the R2 score can be optimistic or pessimistic, depending if it is larger or smaller than the true value. Whatever the order, its accuracy is smaller than if we used all the available data.

My conclusion is that the current sentence is correct.

Reply:
```suggestion
set, so that the mean squared error evaluated on the test set can be pessimistic (because the 
metamodel is trained with less data) or optimistic (because the R2 score has a greater variability).
```